Palindrome/palindrome_no.c

```diff
@@ -0,0 +1,39 @@
+#include<stdio.h>
+int main()
+{
+	int n,rem,temp,rev=0;
+	printf("\nEnter a number: ");
+	scanf("%d",&n);
+	temp=n;
+	while(temp!=0)
+	{
+		rem=temp%10;

```

Comment:
Please follow 4 space indentation across the file. Here, it seems to have a "Tab"

Reply:
@MastersAbh  can you please tell me how to do so??

Reply:
Instead of tab as an indentation, please add 4 spaces for each block